src/main/java/com/minecolonies/colony/CitizenData.java

```diff
@@ -429,7 +429,6 @@ public AbstractJob getJob()
         }
         catch(ClassCastException exc)
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1166)